source/manuals/programming-languages/html.html.md.erb

```diff
@@ -1,85 +1,162 @@
 
-* Always quote attribute values - the decision tree for remember when to
-  selectively quote is too much mental overhead, so just quote them
-* Always use well-formed HTML - implicitly closed elements such as
-  `<p>` cause unnecessary parsing overhead and can cause issues in
-  accessibility
+The GOV.UK Service Manual lists the [minimum browsers that GOV.UK services are required
+to support](https://www.gov.uk/service-manual/technology/designing-for-different-browsers-and-devices#browsers-to-test-in).
+Tools such as GOV.UK Frontend, that are used to help build services, may have
+[stricter standards of browser support](https://github.com/alphagov/govuk-frontend#browser-support).
+Therefore, you will need to test your code in different browsers depending on
+which part of GDS you are working in.
 
-## Use [ARIA](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA) roles appropriately
+Whatever your level of browser support, HTML should be written in such a way
+that older browsers see an appropriate fallback or warning message. For example,
+some browsers do not support the `<video>` element, so you should ensure that
+some contextual information is included (the `<p>` element in this example):
 
-### Guidance on using [ARIA](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA) landmarks
+```html
+<video width="320" height="240" controls>
+  <source src="movie.mp4" type="video/mp4">
+  <source src="movie.ogg" type="video/ogg">
+  <p>Unable to load video. Please upgrade your browser.</p>
+</video>
+```
 
-When you use HTML5 sectioning elements it's a good idea to include
-the appropriate [ARIA](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA) landmark roles as well. The trick when you're
-using both HTML5 and [ARIA](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA) is to put the landmark on the sectioning
-element itself because this avoids the information being duplicated by
-screen readers that support both technologies.
+When using HTML5 elements such as `<main>`, ensure that you include a
+[shim that allows them to be styled](https://github.com/aFarkas/html5shiv) in
+legacy browsers. Include the script inside a `<!--[if lt IE 9]><![endif]-->`
+conditional comment so that modern browsers do not download them.
 
-Example:
+Your page should work with just HTML, before adding any CSS or JavaScript. Read
+[building a resilient frontend using progressive enhancement](https://www.gov.uk/service-manual/technology/using-progressive-enhancement).
+
+## Document structure
 
-    <header role="banner"></header>
+Use HTML5 to structure your page semantically. Use ARIA roles to help
+older assistive technologies map the sections correctly.
 
-The following roles map directly to HTML5 sectioning elements:
+Léonie Watson explains [why using ARIA landmark roles in this way delivers the
+best experience for screen reader users](http://tink.uk/screen-readers-aria-roles-html5-support/).
```

Comment:
Off-topic for this PR, but it would be good to know which browser/AT combination needs that extra support so we know when we can remove it.

Reply:
Agreed 👍